examples/src/main/scala/com/criteo/cuttle/examples/HelloWorld.scala

```diff
@@ -45,35 +45,49 @@ object HelloWorld {
           val partitionToCompute = (e.context.start) + "-" + (e.context.end)
 
           // The `sh` interpolation is provided by the local platform. It allows us to
-          // declare a bash script to execute.
-          sh"""
-          echo "Hello for ${partitionToCompute}"
-          echo "Check my project page at https://github.com/criteo/cuttle"
-          sleep 1
-        """.exec()
+          // declare a bash command to execute. If you want use many commands you can combine them
+          // in a for comprehension.
+          for {
+            _ <- sh"echo Hello for $partitionToCompute".exec()
+            _ <- sh"echo Check my project page at https://github.com/criteo/cuttle".exec()
+            completed <- sh"sleep 1".exec()
```

Comment:
so by the way `sh` name is not relevant anymore.

We could change to something like:

```
exec"sleep 1"()
```

Reply:
```sh"sleep 1".exec()``` becomes ```exec"sleep 1"()```